src/config.rs

```diff
@@ -166,6 +171,26 @@ impl OrchestratorConfig {
             warn!("no chunker configs provided");
         }
 
+        if config.allowed_headers_passthrough.is_none() {
+            warn!("No allowed headers specified");
```

Comment:
```suggestion
            info!("No allowed headers specified");
```
I'm not super versed in logging levels, but should this be `info` instead of `warning`?

Reply:
Hm, I think `info` is also fine for this one.